examples/react-quickstart/README.md

```diff
@@ -14,17 +14,17 @@ To learn more about XMTP and get answers to frequently asked questions, see [FAQ
 
 ### Limitations
 
-This example app demonstrates features of the React SDK, but it isn't a complete solution. For example, the list of conversations doesn't update when new messages arrive in existing conversations.
+This example app demonstrates features of the React XMTP client SDK, but it isn't a complete solution. For example, the list of conversations doesn't update when new messages arrive in existing conversations.
 
 ## Developing
 
-After installing the dependencies, run `yarn dev` to start developing.
+Install the [React client SDK and requirements](/packages/react-sdk/README.md) and then run `yarn dev` to start developing.
```

Comment:
it's not so much "install" as it is "build" that needs to occur prior to running `yarn dev` inside of this folder. running `yarn dev` at the repo root will handle this automatically.

to build manually, you need to go to the `react-sdk` folder and run `yarn build` first.